numpy/lib/function_base.py

```diff
@@ -4005,11 +4005,14 @@ def _trapz_dispatcher(y, x=None, dx=None, axis=None):
 
 @array_function_dispatch(_trapz_dispatcher)
 def trapz(y, x=None, dx=1.0, axis=-1):
-    """
+    r"""
     Integrate along the given axis using the composite trapezoidal rule.
 
-    Integrate `y` (`x`) along given axis.
-
+    If `x` is provided, the integration happens in sequence along its elements - they are not sorted.
```

Comment:
This comment feels out of place here - maybe move it down to the parameters section?